bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -79,7 +79,8 @@ export class Wallet {
      * Destroy the Wallet and drop its database connection.
      */
     async destroy(): Promise<void> {
-        await this.methodHandler.destroy();
+        await (await this.getClient()).destroy();
```

Comment:
```suggestion
```
Makes no sense to create a ClientMethodHandler just to destroy the Client inside again

Reply:
i wasnt sure about this, but youre right